INN/umbrella-caribbean
# Add an empty element to the page to get the article border bottom with logo

## Changes

This pull request makes the following changes:

- outputs an empty div after the end of stories
- styles that div to appear as a black line with the site logo

<img width="1222" alt="Screen Shot 2019-11-26 at 20 41 47" src="https://user-images.githubusercontent.com/1754187/69686306-663e1800-108d-11ea-90e7-5b0235f1563d.png">
<img width="378" alt="Screen Shot 2019-11-26 at 20 41 54" src="https://user-images.githubusercontent.com/1754187/69686307-663e1800-108d-11ea-8717-05b1a8321112.png">


## Why



For #23

## Testing/Questions

Features that this PR affects:

- article pages


Questions that need to be answered before merging:

- [ ] bigger or smaller?
- [ ] if they change the site logo in the future, do we need to draw this image from the theme settings in order to make it easily updatable?

Steps to test this PR:

1.  Check out the PR
2. Visit an article page.
